src/components/atoms/Timeline/index.tsx

```diff
@@ -0,0 +1,244 @@
+import { FC, memo, ReactElement, useMemo } from "react";
+
+import { styled } from "@reearth/theme";
+
+type Range = {
+  start: number;
+  end: number;
+};
+
+export type Props = {
+  /**
+   * @description
+   * This value need to be epoch time.
+   */
+  currentTime: number;
+  /**
+   * @description
+   * These value need to be epoch time.
+   */
+  range?: { [K in keyof Range]?: Range[K] };
+  renderKnob: () => ReactElement;
+  knobSize: number;
+};
+
+const EPOCH_SEC = 1000;
+const PADDING_HORIZONTAL = 12;
+const STRONG_MEMORY_HOURS = 3;
+const STRONG_MEMORY_WIDTH = 2;
+const NORMAL_MEMORY_WIDTH = 1;
+const GAP_HORIZONTAL = 14;
+const HOURS_SECS = 3600;
+const MEMORY_INTERVAL = 600;
+const DAY_SECS = 86400;
+
+const getRange = (_range: Props["range"]): Range => {
```

Comment:
Please tell me if I'm wrong, but are you doing the _XX to denote privacy (the Underscore convention)? 

Since it has no actual affect in JavaScript I think it is better not to use this convention in our source code. The reason being is our code is open source, and I hope our code-base is as accessible to the OSS community as possible. This includes people who are coming from other programming languages and might know only rudimentary JS. Seeing something like this might lead them to believing it is syntactically private, not just conveying that it is private to the developer.

If you disagree and believe it is better to write in this way please tell me and we can discuss!

Reply:
I fixed!
[chore: remove _ prefix](https://github.com/reearth/reearth-web/pull/256/commits/676d96cba6b08f2e7619a38af8433487aec7602b)